addons/hr_holidays/static/scss/hr_leave_mobile.scss

```diff
@@ -1,6 +0,0 @@
-@include media-breakpoint-down(sm) {
-    .o_hr_holidays_dates {
-        display: flex;
-        flex-flow: column;
```

Comment:
This is different from `.o_row`, I think? But why can we remove it?

Reply:
flex-flow: column; was a non elegant way to avoir to set flex properties on children... Width o_row class, children css properties are correctly set.